en/docs/services/service-concepts.md

```diff
@@ -1,46 +1,43 @@
 # Service Concepts
 
-Before you start creating services with Choreo, there are basic concepts that you need to understand. This section walks you through all the service concepts you need to know.
+Choreo services allows you to create enterprise integration services incorporating complex business logic in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo services as you develop your services in the [Choreo Console]({{choreo_console}}).
```

Comment:
```suggestion
Choreo services allow you to create enterprise integration services incorporating complex business logic in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo services as you develop your services in the [Choreo Console]({{choreo_console}}).
```

Reply:
The verb is correct actually. It's because the first two words are proper nouns. However, the second word in the proper noun needs to be capitalized.  I was thinking because we do get some proper nouns that do not require second word to be capitalized , it applied to every instance. For e.g. "Ice cream".  Fixed it.